Browser_IDE/package-lock.json

```diff
@@ -9,11 +9,22 @@
       "version": "1.0.0",
```

Comment:
And if none of the changes in this file are deliberate, please revert them too, thanks! 😀